indico/modules/events/registration/notifications.py

```diff
@@ -27,15 +28,18 @@ def notify_invitation(invitation, email_subject, email_body, from_address):
 
 def _notify_registration(registration, template, to_managers=False):
     from indico.modules.events.registration.util import get_ticket_attachments
-    attachments = None
+    attachments = []
     regform = registration.registration_form
     tickets_handled = values_from_signal(signals.event.is_ticketing_handled.send(regform), single_value=True)
     if (not to_managers and
             regform.tickets_enabled and
             regform.ticket_on_email and
             not any(tickets_handled) and
             registration.state == RegistrationState.complete):
-        attachments = get_ticket_attachments(registration)
+        attachments += get_ticket_attachments(registration)
+    if (not to_managers and registration.registration_form.complete_registration_attach_ical):
```

Comment:
```suggestion
    if not to_managers and registration.registration_form.complete_registration_attach_ical:
```

I'm surprised flake8 didn't complain about this :/